Monthly_Reports/Data_Usage/monthly_usage_report.Rmd

```diff
@@ -128,90 +128,91 @@ transpose_storage %>%
   ) %>%
   tab_footnote(
     footnote = 'Includes preprint supp nodes',
-    locations = cells_data(
+    locations = cells_body(
       columns = vars(Type),
       rows = c(2, 5)
     )
   ) %>%
   tab_footnote(
     footnote = 'After consult with product, no deleted files [even ones > 30 days old] included in total starting 9/20. Before 9/20, deleted storage is included in Total, but not listed in the table',
-    locations = cells_data(
+    locations = cells_body(
       columns = vars(Type),
       rows = c(1, 4)
     )
   )
 
-
 ```
 
 \
 \
 \
 
-```{r, warning=FALSE, echo=FALSE}
+```{r monthly storage table, warning=FALSE, echo=FALSE}
 # monthly storage
+# cells_data() is deprecated, changed to cells_body()
 transpose_storage %>% 
-  filter(Type == 'Nodes' | Type == 'Registrations' | Type == 'Preprints' | Type == 'Quickfiles') %>%
+  filter(Type == 'Nodes' | Type == 'Registrations' | Type == 'Preprints') %>%
   gt() %>%
   cols_align(align = c('center')) %>%
   tab_header(
     title = 'Storage by Product'
   ) %>%
   tab_row_group(
     group = 'Monthly Total',
-    rows = 1:4
+    rows = 1:3
   ) %>%
   tab_row_group(
     group = 'Monthly Increase',
-    rows = 5:8
+    rows = 4:6
```

Comment:
Just to confirm, should the second index be decreased by 2 rather than 1? I only ask b/c most others have gone down by one.

Reply:
Thanks for checking! I believe the 4:6 index is correct. Once the Quickfiles were removed from this table, the first 3 rows (1:3) of the table are used for the 3 types (Total, Nodes, Registrations) in the Monthly Total section and similarly, the next 3 rows (4:6) are used for the Monthly Increase section. Previously, because Quickfiles was included, the first 4 rows (1:4) went to Monthly Total and the next 4 (5:8) went to Monthly Increase